supabase/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "@nawadi/supabase",
   "scripts": {
-    "start": "supabase start",
+    "start": "supabase db start",
```

Comment:
Maybe rename this script to make it more explicit that it only starts the DB since one might already use this to spin up a complete locale development environment (with auth, storage, etc.).